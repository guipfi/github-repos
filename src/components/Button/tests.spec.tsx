import { Button } from '.';
import renderer from 'react-test-renderer';

describe('button tests', () => {
  it('should render a button with text', () => {
    const component = renderer.create(<Button>Button example</Button>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
