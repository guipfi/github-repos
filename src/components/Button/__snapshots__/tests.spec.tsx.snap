// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button tests should render a button with text 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Array [
        undefined,
        undefined,
      ]
    }
  >
    <Text>
      Button example
    </Text>
  </View>
</View>
`;
